staging/2018/benchmarks/csv/000_benchmarks_base.csv

```diff
@@ -160,9 +160,6 @@ Functional Status Change for Patients with General Orthopaedic Impairments,223,R
 Melanoma: Overutilization of Imaging Studies in Melanoma,224,Registry/QCDR,Process,Y,--,--,--,--,--,--,--,100,Yes,
 Radiology: Reminder System for Screening Mammograms,225,Claims,Structure,Y,94.20 - 98.22,98.23 - 99.67,99.68 - 99.99,--,--,--,--,100,No,
 Radiology: Reminder System for Screening Mammograms,225,Registry/QCDR,Structure,Y,--,--,--,--,--,--,--,100,Yes,
-Preventive Care and Screening: Tobacco Use: Screening and Cessation Intervention,226,Claims,Process,Y,96.65 - 98.59,98.60 - 99.61,99.62 - 99.99,--,--,--,--,100,Yes,
-Preventive Care and Screening: Tobacco Use: Screening and Cessation Intervention,226,EHR,Process,Y,72.72 - 82.40,82.41 - 87.55,87.56 - 90.50,90.51 - 92.74,92.75 - 94.63,94.64 - 96.52,96.53 - 98.51,>= 98.52,No,
-Preventive Care and Screening: Tobacco Use: Screening and Cessation Intervention,226,Registry/QCDR,Process,Y,83.72 - 88.38,88.39 - 91.10,91.11 - 93.34,93.35 - 95.44,95.45 - 97.35,97.36 - 99.00,99.01 - 99.99,100,No,
```

Comment:
there were 4 deleted above but only three here. does there need to be another one deleted

Reply:
missing is cmsWebInterface

Reply:
nope, it looks like this the problem is cmsWebInterface shouldn't have been removed above-- that was added in https://github.com/CMSgov/qpp-measures-data/pull/227/files in the json, but not the csv, so when I reran build-benchmarks these changes were overwritten